pkgs/tools/misc/aptly/default.nix

```diff
@@ -1,24 +1,40 @@
 { stdenv, buildGoPackage, fetchFromGitHub, makeWrapper, gnupg1compat, bzip2, xz, graphviz }:
 
-buildGoPackage rec {
-  name = "aptly-${version}";
+let
+
   version = "0.9.7";
   rev = "v${version}";
 
-  src = fetchFromGitHub {
+  aptlySrc = fetchFromGitHub {
     inherit rev;
     owner = "smira";
     repo = "aptly";
     sha256 = "0j1bmqdah4i83r2cf8zcq87aif1qg90yasgf82yygk3hj0gw1h00";
   };
 
+  aptlyCompletionSrc = fetchFromGitHub {
+    rev = version;
+    owner = "aptly-dev";
+    repo = "aptly-bash-completion";
+    sha256 = "1yz3pr2jfczqv81as2q3cizwywj5ksw76vi15xlbx5njkjp4rbm4";
+  };
+
+in
+
+buildGoPackage {
+  name = "aptly-${version}";
+
+  src = aptlySrc;
+
   goPackagePath = "github.com/smira/aptly";
   goDeps = ./deps.nix;
 
   nativeBuildInputs = [ makeWrapper ];
 
   postInstall = ''
     rm $bin/bin/man
+    mkdir -p $out/etc/bash_completion.d
+    ln -s ${aptlyCompletionSrc}/aptly $out/etc/bash_completion.d
```

Comment:
Can you change to `$out/share/bash-completion/completions/` please as it's more common, also look: https://github.com/NixOS/nixpkgs/issues/21053 currently installing this package on nixos would not include bash completions, as for go packages only bin is installed by default, we need something like completions output, i'm also facing the same problem.